ui/src/components/visualizations/MapLibre/index.tsx

```diff
@@ -89,15 +89,15 @@ const MapLibre: React.FC<Props> = ({ className, fileContent, fileType }) => {
             onSelectedFeature={setSelectedFeature}
           />
         )}
+        {/* <FullscreenControl position="top-right" /> */}
+        {selectedFeature && (
+          <MapSidePanel
+            selectedFeature={selectedFeature}
+            setSelectedFeature={setSelectedFeature}
+            onFlyToSelectedFeature={handleFlyToSelectedFeature}
+          />
+        )}
```

Comment:
If you are purposely putting this in the Map as a child, remove the parent div's relative class (also the div might not be needed anymore)